packages/libs/react-ui/src/components/Form/Input/Input.tsx

```diff
@@ -32,24 +30,17 @@ export interface IInputProps
 
 export const Input: FC<IInputProps> = forwardRef<HTMLInputElement, IInputProps>(
   function Input(
-    {
-      outlined,
-      leadingText,
-      leadingTextWidth,
```

Comment:
LeadingTextWidth should still be able to be applied via the InputProps as well. Additionally, it should override the leadingTextWidth provided by the context if both are configured